velox/expression/tests/utils/ArgumentTypeFuzzer.cpp

```diff
@@ -35,6 +38,34 @@ std::optional<TypeKind> baseNameToTypeKind(const std::string& typeName) {
   return tryMapNameToTypeKind(kindName);
 }
 
+void ArgumentTypeFuzzer::determineUnboundedIntegerVariables() {
+  for (const auto& arg : signature_.argumentTypes()) {
+    if (arg.baseName() == "decimal" || arg.baseName() == "DECIMAL") {
+      auto precision = arg.parameters()[0].baseName();
+      auto scale = arg.parameters()[1].baseName();
```

Comment:
> Sometimes precision and scale does not come in pair. I wonder if this works for case like decimal(precision, 6) or decimal(12, scale).

@rui-mo Rui, I saw this question in the e-mail, but for some reason can't find it on GitHub. 

This logic handles precision and scale parameters independently. It should work for the cases above, but needs to be checked. 

I think we can split the decimal support in Fuzzer into a few pieces.

One piece is to enhance ArgumentTypeFuzzer to fuzz arguments for signatures like between(decimal(p, s), decimal(p, s)) -> boolean. 

Another piece is to enhance ArgumentTypeFuzzer to fuzz decimal return types: e.g. least(decimal(p, s),...) -> decimal(p, s), floor(decimal(p, s)) -> decimal(rp, 0).

Reply:
Thanks!